### PR TITLE
Moves some contents from shared engineering storage

### PR DESCRIPTION
**Pull request by 81Denton**
:cl: Denton
tweak: The EngiVend, welding and electrical lockers in Meta/Delta shared engineering storage have been moved back into the Engineering department. The reason for this is that Atmos techs never had ID access to them in the first place.
/:cl:

So uhh when I opened PRs #39144 and #39193, I was under the impression that Atmos techs always had access to electrical/welding lockers and EngiVend - that's not the case, I mixed up regular access permissions and skeleton crew access.

Since Atmos techs have no access to use them, I moved both lockers and the EngiVend back to Engineering. The lathe/printer/YouTool are still there; I also added an extra tool locker.

tl;dr I can't read

**Meta**

meta

**Delta**

delta

### CODE REVIEW

**Comment by 81Denton** [comment]
Thanks! Kinda makes me wish that we had bench sprites, so I can turn them into proper locker rooms.

**Comment by Barhandar** [comment]
And I don't like the new Meta locker room for reasons of "you moved the RnD setup so it needs additional movement to get to from the door". But eh, too late.
